go.local.mod

```diff
@@ -5,8 +5,8 @@ go 1.14
 replace github.com/aws-controllers-k8s/runtime => ../runtime
 
 require (
-	github.com/aws-controllers-k8s/runtime v0.7.1
-	github.com/aws/aws-sdk-go v1.37.4
+	github.com/aws-controllers-k8s/runtime v0.8.0
```

Comment:
go.local.mod changes to runtime are ignored (because of the `replace` line above...)